docs-parts/queries/03-Fetch_lang1.rst

```diff
@@ -35,6 +35,25 @@ Primary key values
 
 ``KEY`` can also used when returning attribute values as separate variables, such that one of the returned variables contains the entire primary keys.
 
+Sorting and limiting the results
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To sort the result, add the additional ``order_by`` argument to ``fetch`` .
+
+.. code-block:: python
+
+    data = query.fetch(order_by='name')
```

Comment:
```suggestion
    data = query.fetch(order_by='name')
    data = query.fetch(order_by='name desc').   # sort in descending order
    data = query.fetch(order_by=('name desc', 'year')).   # by name first, year second 
    data = query.fetch(order_by='KEY')  #  sort by the primary key
    data = query.fetch(order_by=('name', 'KEY desc')).  # sort by name but for same names order by primary key
```

Reply:
added in slightly different format / without traililng '.' in 0b614fa